linux_os/guide/system/auditing/policy_rules/audit_basic_configuration/rule.yml

```diff
@@ -0,0 +1,77 @@
+documentation_complete: true
+
+prodtype: rhel8
+
+title: 'Configure basic parameters of Audit system'
+
+{{% set file_contents_audit_base_config =
+'## First rule - delete all
+-D
+
+## Increase the buffers to survive stress events.
+## Make this bigger for busy systems
+-b 8192
+
+## This determine how long to wait in burst of events
+--backlog_wait_time 60000
+
+## Set failure mode to syslog
+-f 1
+
+' %}}
```

Comment:
The syntax of this Jinja variable can be finicky. I can move this to a separate file, but then, the contents won't be as much as visible.